src/store/lookup.ts

```diff
@@ -218,7 +218,7 @@ async function lookupCardInStock(store: Store, page: Page, link: Link) {
 }
 
 export async function tryLookupAndLoop(browser: Browser, store: Store) {
-	if (!browser.isConnected()) {
+	if (!browser || !browser.isConnected()) {
```

Comment:
`stopAndExit` sets `browser` to undefined. Adding this check here to gracefully exit this loop.

Reply:
Oh! I think this could help #1174. Not sure though, haven't looked into in depth.